.travis.yml

```diff
@@ -97,7 +97,12 @@ install:
   # Create the devel/setup.bash (run catkin_make with an empty workspace) and
   # source it to set the path variables.
   - git clone https://github.com/gsilano/CrazyS.git
+  - cd ~/catkin_ws/src/CrazyS
+  - git checkout hotfix/build-process
+  - cd ~/catkin_ws/src
   - git clone https://github.com/gsilano/mav_comm.git
```

Comment:
Please, remove in the master (it is only necessary when you want to build the branched version).
 ```
- cd ~/catkin_ws/src/CrazyS
- git checkout hotfix/build-process
- cd ~/catkin_ws/src
 ```